CMP1903M A01 2223/Pack.cs

```diff
@@ -73,54 +74,54 @@ public static bool shuffleCardPack(int typeOfShuffle)
             }
             else
             {
-                return false;
+                return false; // if there is any integer passed through other than the three corresponding to the shuffles it will return false
             }
         }
         // This method will deal one card
         public static Card Deal()
         {
-            if (pack.Count == 0)
+            if (pack.Count == 0) // checks if the pack has been built
             {
-                Console.WriteLine("The pack is empty");
-                return null;
+                Console.WriteLine("The pack is empty"); // if the pack is empty will write to the console that the pack is empty
+                return null; // indicates nothing has been reurned and ends
             }
             else
             {
-                Card FirstCard = pack[0];
-                pack.RemoveAt(0);
-                return FirstCard;
+                Card FirstCard = pack[0]; // Defines Firstcard as the first item in the pack
```

Comment:
perhaps implement an additional feature where you can remove from the other end of the pack instead of from the start